app/components/DuringIcoCountdown.tsx

```diff
@@ -78,6 +79,61 @@ export const HexagonText = (props: IMoneyProps) => {
   );
 };
 
+export const AfterIcoHexagonText = (props: IMoneyProps) => {
+  const { loading, raised, neuMarkAmount, neuMarkToEtherRatio, investorsAccountCreated } = props;
+  if (loading) {
+    return <LoadingIndicator className={styles.loadingIndicator} />;
+  }
+
+  return (
+    <div className={styles.text}>
+      <p className={styles.title}>We have raised an equivalent of: </p>
+      <p className={styles.content} data-test-id="during-ico-total-funds">
+        <MoneyComponent
+          tokenType={TokenType.EURO}
+          decimalPlaces={2}
+          value={raised.mul(config.contractsDeployed.euroEthRate)}
```

Comment:
value should be `COMMITMENT_STATUS_EUR_COMMITTED + COMMITMENT_STATUS_ETH_COMMITTED*EUR_ETH_RATE`